### PR TITLE
Edu 6827 sent offers

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [x] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

Added a new method to an endpoint.

### CODE REVIEW

File: VTEX - Marketplace APIs.json [hunk 1]
```diff
@@ -3424,7 +3424,164 @@
...
+                                      "type": "string",
+                                      "description": "Corresponds to the 3-digit [affiliate](https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187) identification code created by the seller."
+                                    },
+                                    "salesChannel": {
+                                      "type": "string",
+                                      "description": "Sales channel (or [trade policy](https://help.vtex.com/en/tutorial/como-funciona-uma-politica-comercial--6Xef8PZiFm40kg2STrMkMV#master-data)) used in the integration."
+                                    },
+                                    "channelName": {
+                                      "type": "string",
+                                      "description": "Name of the marketplace the feed is about."
```
[the remark this answers is not in the document]
Comment (jumeyermattosarruda) [reply VTEX - Marketplace APIs.json:3518]: E acho que 

> "Name of the marketplace the feed is about."

não é tão preciso gramaticalmente. Talvez:
Name of the marketplace, or channel, present in the feed.
Já que o feed não é só sobre um marketplace necessariamente, mas o marketplace é um objeto dentro do feed.

File: VTEX - Marketplace APIs.json [hunk 2]
```diff
@@ -3424,7 +3424,164 @@
...
+                                    },
+                                    "channelLogo": {
+                                      "type": "string",
+                                      "description": "Graphic symbol that identifies the marketplace."
```
Comment (jumeyermattosarruda) [inline VTEX - Marketplace APIs.json:3522]: URL redirecting to the image file of the graphic symbol that identifies the marketplace*

**Review by jumeyermattosarruda — COMMENTED** [review]
Revisado!

File: VTEX - Marketplace APIs.json [hunk 3]
```diff
@@ -3424,7 +3424,164 @@
...
+                  "Sent Offers"
+                ],
+                "summary": "List Feeds",
+                "description": "In [Sent Offers](https://help.vtex.com/en/tutorial/offers-listing--7MRb9S78aBdZjFGpbuffpE), a feed is a list of information about seller's offers sent to a marketplace, and it is updated in real-time with content provided by the marketplace and VTEX modules. \n\nSince a seller can have integrations with multiple marketplaces, numerous feeds can be associated with an account. This endpoint retrieves a list with all the feeds related to an account.\n\nThe feature is available for integrations with [Mercado Libre Classic](https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq), [Mercado Libre Premium](https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq/51oWBHvVxSs8eAwLQhSbSd), [Netshoes](https://help.vtex.com/pt/tracks/configurar-integracao-da-netshoes--5Ua87lhFg4m0kEcuyqmcCm), and VTEX marketplaces. For more information, check out the [Sent Offers Integration Guide](https://developers.vtex.com/vtex-rest-api/docs/sent-offers-integration-guide-connectors)",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:3433]:
```suggestion
                "description": "In [Sent Offers](https://help.vtex.com/en/tutorial/offers-listing--7MRb9S78aBdZjFGpbuffpE), a feed is a list of information about seller's offers sent to a marketplace, and it is updated in real-time with content provided by the marketplace and VTEX modules. \n\nSince a seller can have integrations with multiple marketplaces, numerous feeds can be associated with an account. This endpoint retrieves a list with all the feeds related to an account.\n\nThe feature is available for integrations with [Mercado Libre Classic](https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq), [Mercado Libre Premium](https://help.vtex.com/pt/tracks/configurar-integracao-do-mercado-livre--2YfvI3Jxe0CGIKoWIGQEIq/51oWBHvVxSs8eAwLQhSbSd), [Netshoes](https://help.vtex.com/pt/tracks/configurar-integracao-da-netshoes--5Ua87lhFg4m0kEcuyqmcCm), and VTEX marketplaces. For more information, check out the [Sent Offers Integration Guide](https://developers.vtex.com/vtex-rest-api/docs/sent-offers-integration-guide-connectors).",
```

File: VTEX - Marketplace APIs.json [hunk 4]
```diff
@@ -3424,7 +3424,164 @@
...
+                                      "type": "string",
+                                      "description": "The `feedId` attribute that identifies a feed between a seller and a channel, follows a standardized pattern that will be used by connectors when calling this endpoint. It follows the pattern `vendor.channel`."
+                                    },
+                                    "affiliateId": {
+                                      "type": "string",
+                                      "description": "Corresponds to the 3-digit [affiliate](https://help.vtex.com/en/tutorial/configuring-affiliates--tutorials_187) identification code created by the seller."
+                                    },
+                                    "salesChannel": {
+                                      "type": "string",
+                                      "description": "Sales channel (or [trade policy](https://help.vtex.com/en/tutorial/como-funciona-uma-politica-comercial--6Xef8PZiFm40kg2STrMkMV#master-data)) used in the integration."
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:3514]:
```suggestion
                                      "description": "Sales channel (or [trade policy](https://help.vtex.com/en/tutorial/how-trade-policies-work--6Xef8PZiFm40kg2STrMkMV#master-data)) used in the integration."
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**